pyquil/device/_specs.py

```diff
@@ -135,6 +135,26 @@ def fCZs(self):
         """
         return {tuple(es.targets): es.fCZ for es in self.edges_specs}
 
+    def fISWAPs(self):
+        """
+        Get a dictionary of ISWAP fidelities (normalized to unity) from the specs,
+        keyed by targets (qubit-qubit pairs).
+
+        :return: A dictionary of ISWAP fidelities, normalized to unity.
+        :rtype: Dict[tuple(int, int), float]
+        """
+        return {tuple(es.targets): es.fISWAP for es in self.edges_specs}
+
+    def fXYs(self):
+        """
+        Get a dictionary of XY(pi) fidelities (normalized to unity) from the specs,
+        keyed by targets (qubit-qubit pairs).
+
+        :return: A dictionary of XY/2 fidelities, normalized to unity.
+        :rtype: Dict[tuple(int, int), float]
+        """
+        return {tuple(es.targets): es.fXY for es in self.edges_specs}
+
```

Comment:
what about standard errors?

Reply:
std errors is a question for someone like @deanna-abrams , who has a better understanding of what's going to be put into the specs database

Reply:
yeah, we should get errors for XY(pi) the same way we do for CZ.